rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -128,4 +150,52 @@ private Config getNetworkDefaultConfig(Config userConfig) {
         logger.info("Network not set, using mainnet by default");
         return ConfigFactory.load(MAINNET_RESOURCE_PATH);
     }
+
+    private static void verify(String key, @Nullable ConfigValue expectedValue, ConfigValue actualValue) {
+        if (expectedValue == null) {
+            throw unexpectedKeyException(key, actualValue);
+        }
+
```

Comment:
Isnt a way to compare types before going to the switch, it seems redundant to me, asking in each switch statement for the equal type. Whats the logic there?

Reply:
I've updated the code a bit.